superbench/analyzer/data_diagnosis.py

```diff
@@ -326,6 +326,7 @@ def output_diagnosis_in_json(self, data_not_accept_df, output_path):
         data_not_accept_df['Index'] = data_not_accept_df.index
         data_not_accept_df = data_not_accept_df.rename(
             columns={
+                'Defective Details': 'diagnosis/issue_details',
```

Comment:
Why we have two details?

Reply:
we only have 1 details, but defferent name, this is because previously json format with '--output-all' used Issue_Details while without '--output-all' used 'Defective Details', now I want to unify them to 'diagnosis/issue_details'

Reply:
Yes, but it looks like hacking here?

Reply:
unified these 2 fileds to Defective details and removed this line